<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicensePlateRecognition</class>
 <widget class="QWidget" name="LicensePlateRecognition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车牌识别</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>车牌号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inPath">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="inNumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCho">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="lCar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string>车牌显示</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
